values.yaml

```diff
@@ -198,6 +198,25 @@ global:
       secretName: null
       secretKey: null
 
+  # Settings related to federating with another Consul datacenter.
+  federation:
+    # **Experimental: Currently this is only available in Consul 1.8.0-beta.**
```

Comment:
One thing that would be nice is to say what to set the `global.image` once we know what that image would be.

Reply:
👍